projects/npm-tools/packages/npm-scripts/src/scripts/build.js

```diff
@@ -108,25 +103,13 @@ module.exports = async function (...args) {
 		);
 	}
 
-	const runFederationBuild = federation && federation.mode !== 'disabled';
-
-	if (fs.existsSync('webpack.config.js') || runFederationBuild) {
+	if (fs.existsSync('webpack.config.js')) {
 		webpack(...args);
 	}
 
-	if (runLegacyBuild && bundling !== false) {
+	if (BUILD_CONFIG.bundler) {
```

Comment:
Back before federation, [we used to call `runBundler()` unconditionally](https://github.com/liferay/liferay-frontend-projects/blob/8ff8cb02894f0b6dbcdc9e082e5baca61987ef15/projects/npm-tools/packages/npm-scripts/src/scripts/build.js#L102). Maybe we still should?

Reply:
This is "new", in the sense that we didn't have that before, but since it proved itself useful when I introduced it for webpack federation, I'm leaving it.